controls/std_openeuler.yml

```diff
@@ -0,0 +1,1812 @@
+---
+policy: 'Standard Benchmark for openEuler'
+title: 'Standard Benchmark for openEuler'
+id: std_openeuler
+version: '1.0'
+levels:
+  - id: base
+
+controls:
```

Comment:
Not sure if it is a good idea to create reference IDs containing words as well... Are they named like that in the original security policy? I think numbers could be enough, you can put the description in the "title" tag.

Reply:
I'm sorry, I don't quite understand your question, but you're saying it's not appropriate to use the word "base" in the id field, I can use a number, or something like "level 1", is it right?